libs/core/models/query-params.ts

```diff
@@ -0,0 +1,21 @@
+import { AnimeSort } from './anime-sort';
+import { AnimeType } from './anime-type';
+
+/** Params. */
+export type QueryParams = {
```

Comment:
Please use the prefix of model, because theoretically these may not be the only query parameters in the application.

```suggestion
export type AnimeQueryParams = {
```